cmd/flowlogs2metrics/main.go

```diff
@@ -155,7 +156,7 @@ func run() {
 	)
 
 	// Starting log message
-	log.Infof("starting %s", filepath.Base(os.Args[0]))
+	fmt.Printf("%s starting - version [%s]\n\n", filepath.Base(os.Args[0]), Version)
```

Comment:
Reporting version number on start up (this is seen in the container when we run flowlogs2metrics)
Moving this to be printf so we can always get this even if debug is in error (very useful for Palmetto)